src/statement/mod.rs

```diff
@@ -53,6 +53,15 @@ pub struct Cursor<'a, 'b: 'a, 'c: 'a, S: 'a> {
     buffer: Vec<u8>
 }
 
+#[derive(Debug)]
```

Comment:
Let's derive `Clone` too. Since this type is exposed in the public interface some documentation would be nice.